frontend-vue/locales/en.json

```diff
@@ -271,6 +271,7 @@
   "domains.modal.create_subdomain.title": "Create Subdomain",
   "navigation.domains": "Domains",
   "navigation.search": "Search",
```

Comment:
We can remove that one.

Reply:
Nevermind that. I see that it is still used in the other PR for the new navigation entry `Search`